src/components/Dashboard/Form/DocumentForm.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable prettier/prettier */
 import React, { Component } from "react";
 import FormHeader from "./FormHeader/FormHeader.js";
 import { Form } from "react-final-form";
 import styles from "./DocumentForm.scss";
 import arrayMutators from "final-form-arrays";
 import PropTypes from "prop-types";
+import Button from "../Form/Button/Button";
```

Comment:
I guess Button should be a general component so the file with it could be in components folder instead nested in Form, if it's more specific consider naming it somehow different

@Maciek246 what do you think?

Reply:
Right, all app buttons should look similar so a good idea is to move this component higher.

Reply:
done